src/amazon-sns-sqs-mcp-server/README.md

```diff
@@ -129,25 +129,17 @@ uv run awslabs.amazon-sns-sqs-mcp-server --disallow-resource-creation
 The MCP server implements a security mechanism that only allows modification of resources that were created by the MCP server itself. This is achieved by:
 
 1. Automatically tagging all created resources with a `mcp_server_version` tag
-2. Validating this tag before allowing any mutative actions (update, delete, reboot)
+2. Validating this tag before allowing any mutative actions (update, delete) - this is a deterministic check that ensures only resources created by the MCP server can be modified
 3. Rejecting operations on resources that don't have the appropriate tag
-4. A2P Mutative Operations are not enabled
+4. [Application-to-Person](https://docs.aws.amazon.com/sns/latest/dg/sns-user-notifications.html) (A2P) messaging mutative operations are not enabled by default for security reasons
 
 ## Best Practices
 
-- Use descriptive broker names to easily identify resources
+- Use descriptive topic and queue names to easily identify resources
 - Follow the principle of least privilege when setting up IAM permissions
 - Use separate AWS profiles for different environments (dev, test, prod)
-- Monitor broker metrics and logs for performance and issues
 - Implement proper error handling in your client applications
 
-## Security Considerations
-
-When using this MCP server, consider:
-
-- The MCP server needs permissions to create and manage Amazon SNS / SQS resources
-- Only resources created by the MCP server can be modified by it
-
```

Comment:
Consider keeping this and referencing when the argument allows resource changes.